pwa/components/dashboard/repairers/RepairersList.tsx

```diff
@@ -62,7 +62,11 @@ const RepairersList = ({repairers}: RepairersListProps) => {
                 ) : (
                   <>
                     <Link
-                      href={`/sradmin/boutiques/${repairer.id}/edit`}
+                      href={
+                        repairer.enabled
+                          ? `/sradmin/boutiques/${repairer.id}/edit`
+                          : `/sradmin/boutiques/${repairer.id}`
+                      }
```

Comment:
Is it requested to disabled edition if the repairer is not enabled ? Because it can be useful to edit a no enabled repairer if the user has made a mistake

Reply:
In the backend, a non enabled repairer cannot have access to PUT or PATCH, it was to be consistent.

Reply:
Ok, I didn't remembered that, you can keep as currently